application/src/main/java/org/opentripplanner/graph_builder/module/osm/EscalatorProcessor.java

```diff
@@ -27,30 +34,41 @@ public void buildEscalatorEdge(OsmWay escalatorWay, double length) {
       .boxed()
       .toList();
 
+    Optional<Duration> duration = escalatorWay.getDuration();
+    if (duration.isPresent()) {
+      double speed = length / duration.get().toSeconds();
+      if (speed < SLOW_ESCALATOR_ERROR_CUTOFF || speed > FAST_ESCALATOR_ERROR_CUTOFF) {
+        duration = Optional.empty();
```

Comment:
Are you interested in correcting these cases? If so, you can add an issue to the issue store?

Reply:
This is more of a guard against possible insanely wrong tags that I added after a discussion with Joel. I don't know if this has a real world use, but better safe than sorry.